cmd/log-shuttle/metrics_reporter.go

```diff
@@ -36,7 +44,7 @@ func LogFmtMetricsEmitter(r metrics.Registry, source string, d time.Duration, l
 		r.Each(func(name string, i interface{}) {
 			switch metric := i.(type) {
 			case metrics.Counter:
-				ctx[name] = metric.Count()
+				countDifference(ctx, name, metric.Count())
```

Comment:
Previously these would have just been "metric.name", now they will be "metric.name.count". This strikes me as minor, but it is unexpected.